docker/podman-docker.sh

```diff
@@ -0,0 +1,8 @@
+# DOCKER_HOST initialization
+
+[ -z "$DOCKER_HOST" ] || return
+if [ "$EUID" -eq 0 ]; then
+       DOCKER_HOST=unix:///run/podman/podman.sock
+else
+       [ ! -z "$XDG_RUNTIME_DIR" ] && DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
```

Comment:
use `-n` over `! -z`

Also doesn't one have to export these vars to actually make then environment variables?